pkg/client/monitoring/v1alpha1/types.go

```diff
@@ -125,6 +127,16 @@ type PrometheusStatus struct {
 	UnavailableReplicas int32 `json:"unavailableReplicas"`
 }
 
+// LogSpec defines the configured log messages from the Prometheus logger.
+type LogSpec struct {
```

Comment:
Sure thing for confirmation. Looks like Prometheus 2.0 also uses logrus in prometheus/common/log.
```
$ docker run prom/prometheus:v2.0.0-beta.0 -h
usage: prometheus [<flags>]

The Prometheus monitoring server

Flags:
  -h, --help                     Show context-sensitive help (also try
                                 --help-long and --help-man).
      --version                  Show application version.
      --log.level="info"         Only log messages with the given severity or
                                 above. One of: [debug, info, warn, error,
                                 fatal]
      --log.format="logger:stderr"
                                 Set the log target and format. Example:
                                 "logger:syslog?appname=bob&local=7" or
                                 "logger:stdout?json=true"
```